pyartcd/pyartcd/pipelines/update_golang.py

```
@@ -535,16 +536,12 @@ async def run(self):
                     await self._ensure_builder_pullspec_available(published_pullspec)
-                    builder_pullspecs[el_v] = published_pullspec
                 except RuntimeError:
-                    konflux_pullspec = self._get_konflux_builder_pullspec(record.nvr)
-                    _LOGGER.warning(
-                        "Published pullspec not available: %s, falling back to Konflux pullspec: %s",
-                        published_pullspec,
-                        konflux_pullspec,
+                    raise RuntimeError(
+                        f"Published pullspec not available: {published_pullspec}. "
+                        "Ensure shipment MRs are merged before re-running this pipeline."
                     )
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)